v2/datastream-to-spanner/src/test/resources/DataStreamToSpannerLoadTesting/datagenerator-schema.json

```diff
@@ -0,0 +1,6 @@
+{
+  "user_id": "{{integer(-9223372036854775808,9223372036854775807)}}",
+  "first_name": "{{alphaNumeric(460,500)}}",
```

Comment:
Can the user_id generated here fit into user_id int?
Same question for first_name and last_name. Can alphanumeric(460,500) fit into varchar(50)?

Reply:
int can, I changed it for the names